homeassistant/components/sensor/tibber.py

```diff
@@ -35,15 +34,15 @@
 
     tibber_connection = hass.data.get(TIBBER_DOMAIN)
 
-    try:
-        dev = []
-        for home in tibber_connection.get_homes():
+    dev = []
+    for home in tibber_connection.get_homes():
+        try:
             await home.update_info()
-            dev.append(TibberSensorElPrice(home))
-            if home.has_real_time_consumption:
-                dev.append(TibberSensorRT(home))
-    except (asyncio.TimeoutError, aiohttp.ClientError):
-        raise PlatformNotReady()
+        except (asyncio.TimeoutError, aiohttp.ClientError):
+            pass
```

Comment:
The login token is already validated in the component.
So this should only be a temporary network issue.
`update_info` is called again in the `update` function

Reply:
Maybe at least warn the user about it?

Reply:
I decided to keep the exception, and add some more logging. 
Hopefully the library update will solve the issue, otherwise we will at least get better logging.